README.md

```diff
@@ -1,81 +1,27 @@
+## 2. Run the App Locally
 
-`nodejs` and `npm` must be installed on your system.
+Go to the crypto-demo folder in a terminal and run the following: `npm install`
 
-`trades` collection should be present.
+Once all the node modules have been installed, execute `npm start` to start the development server. This starts a local development server on `localhost:<some_port>`. You can log into the app with your Macrometa credentials. You will need to [configure](https://macrometa.com/docs/apps/crypto-trading) the Crypto App backend for the app to work.
 
-# 3. How to run app(UI) locally
-
-> NOTE: This step is just for running the UI locally. The actual app is deployed on an AWS S3 Bucket. For the steps on S3 go to the `How to deploy app on S3` section.
-
-Go to crypto-demo and run the following
-If `node_modules` is not there, execute `npm install`.
-
-Once all the node modules have been installed execute `npm start` to start the development server. This will start a local development server on `localhost:<some_port>`.
-
-# 4. How to deploy app(UI) on S3
-
-If `node_modules` is not there, execute `npm install`.
-
-Go to crypto-demo and run `npm run build`.
-This will create a folder build.
-
-The contents of this `build` folder need to be copied to the S3 bucket.
-
-If using aws cli run `aws s3 cp build s3://<your-s3-bucket-name> --recursive` to recursively copy all files and folders inside the `build` folder to the S3 bucket.
-
-The bucket needs to be public in order for the website to be visible.
-
-A sample `bucket policy` is:
-
-```js
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Sid": "PublicReadGetObject",
-            "Effect": "Allow",
-            "Principal": "*",
-            "Action": "s3:GetObject",
-            "Resource": "arn:aws:s3:::<your-s3-bucket-name>/*"
-        }
-    ]
-}
-```
-
-Now go to the `Properties` tab in the AWS console for this bucket and open `Static website hosting` option. Then select the option `Use this bucket to host a website` and provide `index.html` for both `Index document` and `Error document` text fields. Click on save and the website is now live!
-
-# 5. How to deploy the stream application.
-
-Deploy the stream application on your federation and activate the app. Stream app definition can be found [here](./streamapps/Crypto-Trading-App.md)
-
-Activate the app and you will be able to see the data in the UI.
-
-# 6. Already deployed demo
-
-Go to `http://crypto.gdn1.s3-website-us-east-1.amazonaws.com/`
-
-Log in using your own credentials or the default ones.
-
-Start the Node server locally to see the data on the Graphs.
+We also have a [Crypto Trading App frontend](https://macrometacorp.github.io/tutorial-cryptotrading/) already deployed. You can log in with the credentials we provide to see the app working, or use your own Macrometa account credentials after you have [configured](https://macrometa.com/docs/apps/crypto-trading) the Crypto App backend.
```

Comment:
Still not clear where the credentials are. If they are in the app topic in the docs, that makes no sense, because we aren't using them there, we're using them here.

Also, change the link text from "configured" to something else, preferably the exact title of what it is linking to.